tensorflow/compiler/mlir/tools/kernel_gen/tests/tf_64bit_indexer.mlir

```diff
@@ -0,0 +1,30 @@
+// RUN: kernel-gen-opt %s --split-input-file \
+// RUN:   --tf-64bit-indexer="architectures=sm_123,sm_456 \
+// RUN:   tile-sizes=1,2,3 unroll-factors=3,2,1 max-supported-rank=32 \
+// RUN:   enable-ftz=false index_64bit=false cpu-codegen=false \
+// RUN:   jit_i64_indexed_for_large_tensors=false" | \
```

Comment:
For what we are testing here, I believe you intended to set `jit_i64_indexed_for_large_tensors` to true. Did this test pass?

Reply:
test was tested for output only. It is fixed now.